swingset/src/main/java/com/nqadmin/swingset/utils/SSCommon.java

```diff
@@ -405,6 +410,19 @@ protected void bind() {
 			return;
 		}
 
+		logger.trace(() -> String.format("Column bind succeeded: name=%s, index=%d %s.",
+				boundColumnName, boundColumnIndex, rowSet==null ? ", rowset=null" : ""));
+
+		//
+		// TODO: is this the right place to do this?
+		//       Will this catch any metadata changes
+		//
+
+		// This is used a lot, just get it now.
+		isNullable = RowSetOps.isNullable(rowSet, boundColumnIndex);
+
+		logger.trace(() -> String.format("Column isNullable: %s.", isNullable));
+
```

Comment:
Is this an OK place to catch all metadata changes. Think this covers bind and setRowSet. Anywhere else?

Reply:
I think SSCommon.bind() is the right spot. setBoundColumnName() and setBoundColumnIndex() call bind() if needed.

Reply:
OK. In the future, could generalize and cache more metadata queries, make them lazy. Here's the spot to flush that non-existent cache.